pkg/controller/garbagecollector/metrics.go

```diff
@@ -54,6 +57,13 @@ var (
 			Help:      "Time in microseconds of an item spend in the orphanQueue",
 		},
 	)
+	DeletionToWatchLatency = prometheus.NewSummary(
```

Comment:
(making a struct to hold all things like this, and then passing the struct around, would be more testable than having all of these as globals like this.)